frontend/src/screens/EventDetails.tsx

```diff
@@ -1,21 +1,151 @@
+    const scrollOffSet = useScrollViewOffset(scrollRef);
+    const imageAnimatedStyle = useAnimatedStyle(() => {
+        return {
+            transform: [
+            {
+                translateY: interpolate(
+                    scrollOffSet.value,
+                    [-IMG_HEIGHT,0, IMG_HEIGHT],
+                    [-IMG_HEIGHT / 2,0, IMG_HEIGHT * 0.75]
+                )
+            },
+            {
+                scale: interpolate(
+                    scrollOffSet.value,
+                    [-IMG_HEIGHT,0, IMG_HEIGHT],
+                    [3, 1, 1]
+                )
+            }
+            ]
+        }
+    })
+    // const { setOptions: setNavigationOptions } = useNavigation();
+    // const { params: { eventNumber } } = useRoute<any>();
+    const [eventData,setEventData] = useState({
+        title:"",
+        date:"",
+        location:"",
+        mainImg:"",
+        clubIcon:"",
+        clubName:"",
+        detail:""
+    });
     useLayoutEffect(() => {
-        setNavigationOptions({
-            headerTitle: `Mock Event ${eventNumber}`
+        // setNavigationOptions({
+        //     headerTitle: `Mock Event ${eventNumber}`
+        // })
+        // Fetch here?
+        setEventData({
+            title:"Valorant Tournament",
+            date: "January 20",
+            location: "University of Calgary",
+            mainImg: "",
+            clubIcon: "",
+            clubName: "E-Sports Club",
+            detail: "Step into the exhilarating realm of competition and showcase your tactical prowess at our upcoming local Valorant tournament! Embark on a thrilling journey where precision meets strategy, as teams battle it out for glory and recognition. Unleash your skills in this adrenaline-fueled arena, where every shot fired and every well-executed strategy could be the turning point in your team's ascent to victory. \n Join fellow enthusiasts in a celebration of camaraderie and sportsmanship, and let the electrifying atmosphere of the tournament propel you to new heights. Whether you're a seasoned veteran or a rising star, this is your chance to leave your mark and etch your name in the annals of local Valorant history.\n The stage is set, the competition is fierce, and the glory awaits—seize the opportunity and be part of an unforgettable gaming experience!"
         })
     }, [])
-
+    const navigation = useNavigation<any>();
+    
     return (
-        <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-            <ThemedText>{`Mock Event ${eventNumber}`}</ThemedText>
-        </View>
+        <MainContainer>
+            <HeaderContainer>
+            <AntDesign
+                        style={{ marginTop: "15%", marginLeft: "3%" }}
+                        name="caretleft"
+                        size={24}
+                        color="white"
+                        onPress={() => {navigation.navigate("Login")}}
```

Comment:
all callbacks should be defined outside of the curly braces using
```React.useCallback()```
See these docs for usage and advantages:
https://react.dev/reference/react/useCallback